rocm-smi-lib std::setw not found

Building 7.0.0 on fedora/rawhide with testing enabled has several build errors similar to

/builddir/build/BUILD/rocm-smi-7.0.0-build/rocm_smi_lib-rocm-7.0.0/tests/rocm_smi_test/functional/frequencies_read.cc:96:12: error: ‘setw’ is not a member of ‘std’
   96 |       std::setw(2) << std::right << clk_i_str << ": " <<
      |            ^~~~
/builddir/build/BUILD/rocm-smi-7.0.0-build/rocm_smi_lib-rocm-7.0.0/tests/rocm_smi_test/functional/frequencies_read.cc:54:1: note: ‘std::setw’ is defined in header ‘<iomanip>’; this is probably fixable by adding ‘#include <iomanip>’
   53 | #include "rocm_smi_test/test_common.h"
  +++ |+#include <iomanip>
   54 | 

iomanip needs to be included. This seems to work
sed -i '/#include <string.*/a#include <iomanip>' tests/rocm_smi_test/test_base.h

--- a/projects/rocm-smi-lib/tests/rocm_smi_test/functional/frequencies_read.cc
+++ b/projects/rocm-smi-lib/tests/rocm_smi_test/functional/frequencies_read.cc
@@ -45,6 +45,7 @@
 
 #include <cstdint>
 #include <iostream>
+#include <iomanip>
 #include <string>
 
 #include "gtest/gtest.h"

